packages/elements/src/chart/elements/chart.ts

```diff
@@ -333,8 +327,29 @@ export class Chart extends BasicElement {
     // set global config again in case the CSS font isn't loaded when updating the chart
     this.setGlobalConfig();
     this.decorateColors(chart);
+    this.decorateGridColors(chart);
   };
 
+  /**
+   * Set grid color in beforeUpdate event from Chartjs
+   * @param chart Chart.js instance
+   * @returns {void}
+   */
+  private decorateGridColors(chart: ChartJS): void {
+    for (const scale in chart.scales) {
+      const axis = chart.options?.scales?.[scale] as CartesianScaleOptions;
+      const userGridColor = this.config?.options?.scales?.[scale]?.grid?.color as Color;
```

Comment:
no need to type cast here, right?
```suggestion
      const userGridColor = this.config?.options?.scales?.[scale]?.grid?.color;
```